python/nav/eventengine/alerts.py

```diff
@@ -82,6 +85,8 @@ def __bool__(self):
     def make_alert(self):
         """Generates an alert object based on the current attributes"""
         attrs = {}
+        if self.severity_rules:
+            self.severity = self.severity_rules.evaluate(self)
```

Comment:
Why not do this in `__init__`?

Reply:
I started with that, but for some reason it didn't work out. I was too lazy to figure out why :-P

Reply:
Because the AlertGenerator is mutable, so it doesn't necessarily have all the information needed to evaluate a severity rule until the last moment: When the model object is actually being created for insertion into the database.